### PR TITLE
HACDOCS-264: Add Stonesoup Learn more side panel content

### PR DESCRIPTION
**Pull request by jseseCCS**
[HACDOCS-264](https://issues.redhat.com/browse/HACDOCS-264)

Adding new Learn more side panel content for Add component, Configure component. Gdoc is [here](https://docs.google.com/document/d/1WrcoWHDYForuvvxdeoVwNSNVbSrzbJxPvWgCiRiXC8U/edit?usp=sharing).

**IMPORTANT:** This "Learn more" side panel content is for Stonesoup, not AppStudio.

### CODE REVIEW

File: docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+kind: HelpTopic
```
Comment (rohitkrai03) [inline docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml:1]: Based on the help-topic [readme](https://github.com/RedHatInsights/quickstarts/blob/main/docs/help-topics/README.md) you should create a new directory for every help topic. So there shouldn't be a `stonesoup-learnmore` directory. You should have `stonesoup-add-a-component` directly as the directory.

File: docs/help-topics/stonesoup-learnmore/stonesoup-configure-a-component/stonesoup-configure-components.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Name is an internal name. Title shows up in the UI as a side panel title.
+# Tags to be kept empty for now. Tags will specify where in the app descriptions will be available.
+# Links to be external only. We don't know yet whether referencing to other side panels will be supported but referencing to in-depth docs is expected to be supported.
+# Note that this topic is for LEARN MORE.
+
+- name: stonesoup-add-a-component
```
Comment (rohitkrai03) [inline docs/help-topics/stonesoup-learnmore/stonesoup-configure-a-component/stonesoup-configure-components.yml:6]: The name should be unique.
```suggestion
- name: stonesoup-configure-components
```

File: docs/help-topics/stonesoup-learnmore/stonesoup-configure-a-component/metadata.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+kind: HelpTopic
+name: stonesoup-make-a-component
```
Comment (rohitkrai03) [inline docs/help-topics/stonesoup-learnmore/stonesoup-configure-a-component/metadata.yml:2]:
```suggestion
name: stonesoup-configure-components
```

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml (hunk 1 above)
Comment (jseseCCS) [reply docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml:1]: I added a stonesoup-learnmore dir just like Jake added a stonesoup-whatsnext dir. This is per Danny Philips' instructions.

File: docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
+name: stonesoup-make-a-component
```
[the remark this answers is not in the document]
Comment (jseseCCS) [reply docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml:2]: Did I fix it, Rohit?

[the remark this answers is not in the document]
Comment (jseseCCS) [reply docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml:2]: ohhhhhh I see what you mean. one sec and i'll fix both

File: docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml (hunk 1 above)
Comment (rohitkrai03) [reply docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml:1]: I tried using whats next help topics added by Jake in the UI. It doesn't work and I am guessing this is the main issue.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

**Comment by jseseCCS** [comment]
@rohitkrai03 before i create a new PR to merge the correct import file, can you look at this and make sure i got the formatting right? this should be exactly the same as [the file](https://redhat-internal.slack.com/archives/C02GG6FUXCH/p1675784711118629?thread_ts=1675445143.785149&cid=C02GG6FUXCH) you shared with me yesterday in slack. thank you!

**Pull request merged**